marathon_lb.py

```diff
@@ -72,6 +72,9 @@ def __init__(self, appId, servicePort, healthCheck):
         self.servicePort = servicePort
         self.backends = set()
         self.hostname = None
+        self.proxypath = ''
```

Comment:
Like I commented on the last PR, any reason this doesn't default to None and then checks using `if app.proxypath:` instead of `if app.proxypath = '':`? Same for the other two.